src/components/MultiModal/MultiModalProvider.tsx

```diff
@@ -1,16 +1,44 @@
 /* eslint-disable react-hooks/exhaustive-deps */
 import React, { useContext, useEffect, useMemo, useCallback } from 'react'
 import { useWallet } from '../../providers/Wallet'
-import PropTypes from 'prop-types'
 import { noop } from '@1hive/1hive-ui'
 import { useSteps } from '../../hooks/useSteps'
 
-const MultiModalContext = React.createContext({})
+type State = {
+  currentScreen: any
+  close: () => void
+  direction: any
+  getScreen: any
+  next: any
+  prev: any
+  step: any
+}
+
+const MultiModalContext = React.createContext<State>({
+  currentScreen: null,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  close: () => {},
```

Comment:
can't we define as  `() => void` ?

Reply:
This is not a type. It's an empty function :) The `() => void` is in the definition of the type.